server/api/handlers/message/message.py

```
@@ -0,0 +1,44 @@
+    
+    # Fetch the sender and receiver users based on their usernames
+    sender = users_chat.find_one({"username": sender_username})
+    receiver = users_chat.find_one({"username": receiver_username})
+    
+    if sender and receiver:
+        save_message(sender['_id'], receiver['_id'], message) 
+        return jsonify({"status": "Message sent!"}), 200
+    else:
+        return jsonify({"error": "User not found!"}), 404
+
+def get_messages(username):
```

Ruff

Redefinition of unused `get_messages` from line 9

(F811)